compiler/prelude/jsmapping.js

```diff
@@ -236,6 +236,9 @@ var $internalize = (v, t, recv, seen, makeWrapper) => {
         case $kindFloat64:
             return parseFloat(v);
         case $kindArray:
+            if (v == null) {
+                return t.zero();
```

Comment:
Yeah, the externalized array has lots of issues around being set correctly even when not zero. I added https://github.com/gopherjs/gopherjs/issues/1302 for the assignment since it does the same things, it copies into a temporary internalized copy but never writes to the externalized version.

Since this has other issues around it, I'll change it to throw an exception right now. Then we can rework when we get a chance to work on externalized arrays in general.

Reply:
Oh, I see what you mean. Okay, yeah, in that case I'll change it to a panic so it can't be used even for reading.
Thanks for helping me with this. The jsmapping stuff is so complex, it'll take me awhile to get my head around it (if I ever do).

Reply:
Okay, now internalizing a null externalized array will throw an error similar to:
```
Error: runtime error: can not internalize null as a [3]int
```